itou/www/apply/views/submit_views.py

```diff
@@ -240,9 +240,9 @@ def __init__(self):
 
     def setup(self, request, *args, session_uuid, **kwargs):
         super().setup(request, *args, **kwargs)
-
-        self.apply_session = SessionNamespace(request.session, APPLY_SESSION_KIND, session_uuid)
-        if not self.apply_session.exists():
+        try:
+            self.apply_session = SessionNamespace(request.session, APPLY_SESSION_KIND, session_uuid)
+        except SessionNamespaceInvalid:
```

Comment:
```suggestion
        except SessionNamespaceDoesNotExist:
```
J'avais compris (et je suis plutôt pour) que c'était `SessionNamespaceDoesNotExist` qu'on souhaitait gérer (typiquement un utilisateur qui fait retour arrière à la fin d'un wizard).
`SessionNamespaceInvalid` c'est un petit malin qui s'amuse avec les URLs: pas certain qu'on souhaite vraiment renvoyer une 500 non plus mais peut-être logger le cas.

Reply:
Effectivement pour les logs.
Par contre, renvoyer une 500 parce qu'un utilisateur revient sur l'URL précédente d'un wizard terminé (et dont la session n'existe donc plus) ne me semble pas une bonne idée.

Reply:
Ah vui, `SessionNamespaceInvalid` != `SessionNamespaceInvalidKind` :man_facepalming: 
Mais du coup, j'approuve ton dernier renommage :smiling_face_with_tear: